doc/cygnus-common/installation_and_administration_guide/management_interface.md

```diff
@@ -242,30 +242,43 @@ Response:
 [Top](#top)
 
 ##<a name="section9"></a>`GET /admin/log`
-Gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts).
-
+Gets the logging level.
 ```
-GET http://<cygnus_host>:<management_port>/admin/log
+GET http://<cygnus_host>:<management_port>/admin/log4j
 ```
 
 Responses:
 
 ```
 200 OK
+{"level": "...."}
+```
+
+```
+500 Internal Server Error
 {
-    "log4j": {
-        "appenders": [
-            {
-                "layout": "...",
-                "name": "..."
-            }
-        ],
-        "level": "..."
-    },
-    "success": "true"
+    "error": "..."
 }
 ```
 
+Gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts) given a parameter `verbose` with a true value. `false` value is invalid.
```

Comment:
Instead, if parameterized with `verbose=true` gets the log4j configuration (relevant parts, as the logging level or the appender names and layouts).